crates/dekaf/src/session.rs

```diff
@@ -383,6 +383,11 @@ impl Session {
         &mut self,
         request: messages::FindCoordinatorRequest,
     ) -> anyhow::Result<messages::FindCoordinatorResponse> {
+        // if matches!(self.auth, Some(SessionAuthentication::Redirect { .. })) {
```

Comment:
Was this left commented intentionally?

Reply:
[It's removed in the subsequent commit](https://github.com/estuary/flow/pull/2365#discussion_r2302037509)